java/ql/src/Language Abuse/EmptyMethod.ql

```diff
@@ -0,0 +1,93 @@
+/**
+ * @id java/empty-method
+ * @name Empty method
+ * @description An empty method serves no purpose and makes code less readable. An empty method may
+ *              indicate an error on the part of the developer.
+ * @kind problem
+ * @precision very-high
+ * @problem.severity warning
+ * @tags correctness
```

Comment:
Should we add CWE tags if relevant for quality queries? Such as maybe [CWE-1071](https://cwe.mitre.org/data/definitions/1071.html) for this query?

Reply:
Yes, I think this makes sense. Also, there's some internal discussion to add some other tags too, such as `quality`.